FAQ.md

```diff
@@ -114,7 +111,9 @@ are applied to each Machine object, but not copied to the Node.  In the case of
 MachineSet, these are in the `.spec.template.metadata` field, or just the `.metadata` field when applied to an individual Machine.
 
 ## Adding Taints to Nodes via Machines
-Similar to Annotations and Labels, MachineSets can be used to add Taints to the Machines and Nodes that it creates. To do this you will need to add the Taint information to your MachineSets. **Note** that these Taints will only be applied to the new Machines and Nodes created from the MachineSet, they will not be retroactively applied.
+Similar to Annotations and Labels, MachineSets can be used to add Taints to the Machines and Nodes that it creates. To do this you will need to add the Taint information to your MachineSets. **Note** that these Taints will only be applied to the new Machines and Nodes created from the MachineSet, they will not be retroactively applied unless added directly to the Machine.
+
+If you wish to remove taint from Node, you will need to remove it from the Machine first to prevent it from being reapplied, and then remove it from the node object.
```

Comment:
```suggestion
If you wish to remove a taint from a Node, you will need to remove it from the Machine first to prevent it from being reapplied, and then remove it from the Node object.
```